test/router-view.spec.js

```diff
@@ -155,12 +157,53 @@ describe('router-view', () => {
       })
       .then(done);
   });
+
+  fdescribe('classes as module id', () => {
```

Comment:
I think you should remove the `f` for the final PR

Reply:
oops 😄